tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "compilerOptions": {
+    "rootDir": ".",
+    "target": "ES2022",
+    "module": "NodeNext",
+    "moduleResolution": "NodeNext",
+    "baseUrl": "./",
+    "sourceMap": true,
+    "outDir": "./dist_local",
```

Comment:
Note that the npm run package command uses ncc to compile the index.ts and all its dependencies into a single index.js file inside of the dist folder. In fact there is a github action to confirm that we don't accidentally submit anything else from dist. To make things neat and organized for local development, i have chosen to have the compile to compile to dist_local instead which is on the .gitignore list now. Also note that this ncc really helps us not have to worry about accidentally shipping the tests - so I was able to make the root directory "." and include the tests in the main build and linting.